src/backend/marsha/core/tests/test_models_video.py

```diff
@@ -68,6 +68,6 @@ def test_models_video_is_ready_to_show(self):
 
         # Only when live is running, the video is ready to show
         for live_choice in LIVE_CHOICES:
-            video = VideoFactory(live_state=live_choice[0])
+            video = VideoFactory(live_state=live_choice[0], live_type=RAW)
```

Comment:
You need to add tests to confirm that the constraint is working?
```
with self.assertRaises(IntegrityError):
    VideoFactory(live_state="running", live_type=None)
```

Reply:
Yes I will do it.